google_directory_service.py

```diff
@@ -72,3 +73,21 @@ def GetUsersByGroupKey(self, group_key):
 
     return users
 
+  def SearchForUser(self, user_key):
```

Comment:
Just a nit: would this be better and more consistent as GetUser()?
def GetUser(self, user_key):
  """Get user based on a user key."""